tools/cloud_functions/gcs_event_based_ingest/cloudbuild.yaml

```
@@ -34,6 +34,13 @@ steps:
     - '--cache=true'
     - '--context=dir:///workspace/${_BUILD_DIR}'
   id: 'build-ci-image'
+- name: 'gcr.io/$PROJECT_ID/gcs_event_based_ingest_ci'
```

last build failed because google cloud sdk not installed in this image.
just use [google/cloud-sdk](https://hub.docker.com/r/google/cloud-sdk/)
```suggestion
- name: 'google/cloud-sdk'
```